sonar-project.properties

```diff
@@ -0,0 +1,16 @@
+# must be unique in a given SonarQube instance
+#sonar.projectKey=my:project
```

Comment:
Commented code

Reply:
I'll delete this PR and create a new one